### DISCUSSION TITLE
Minor upgrade from 3.5.1 to 3.5.3

### DISCUSSION THREAD

**Discussion by anupamdialpad** [etcd-io/etcd#14213] [Q&A] [answered]
Hi,

For upgrade from 3.5.1 to 3.5.4 do we need to take any kind of backup for the etcd cluster? Or is the upgrade minor and can be reverted to 3.5.1 easily?

Thank You

**Comment by serathius** [comment, marked as answer]
Etcd patch (v3.5.1 vs v3.5.4) releases should only include bug fixes and are fully compatible. We don't test reverts, however based on etcd backport policies there should not be any fear of reverting.

I would highly recommend upgrading to v3.5.4 as older v3.5 releases have know critical data inconsistency issues.
